test/metabase/lib/drill_thru/zoom_in_timeseries_test.cljc

```diff
@@ -189,24 +189,24 @@
                       (lib/breakout (-> (meta/field-metadata :orders :created-at)
                                         (lib/with-temporal-bucket unit1))))]
         (lib.drill-thru.tu/test-drill-application
-          {:click-type     :cell
```

Comment:
These diffs tripped me up the other day. Spent a few minutes wondering if my Emacs clojure config was busted before realizing it was the existing indentation that was wrong.